dao/src/main/kotlin/tables/ScanResultsTable.kt

```diff
@@ -21,8 +21,6 @@ package org.eclipse.apoapsis.ortserver.dao.tables
 
 import kotlinx.serialization.Serializable
```

Comment:
The DAO class should represent the database model completely, even if some properties are not yet used, therefore I would prefer to drop that commit.